app/src/main/java/com/pajato/android/gamechat/chat/BaseCreateFragment.java

```diff
@@ -53,8 +54,10 @@
 
     // Public instance methods.
 
-    /** Establish the layout file to show that the app is offline due to network loss. */
-    @Override public int getLayout() {return R.layout.fragment_chat_create;}
+    @Override public void onCreate(Bundle bundle) {
```

Comment:
Put the javadoc comment back in.  Feel free to make the comment better but if there is one already available and it is not wrong, don't remove it.